### PR TITLE
Autocomplete dropdown

### PR DESCRIPTION
**Pull request by mellogarrett**
Refactored the Dropdown component to an Autocomplete component using `downshift`

### CODE REVIEW

File: components/dropdown-question/index.tsx [hunk 1]
```diff
@@ -12,44 +17,106 @@ interface DropdownQuestionProps {
...
+                          selected: highlightedIndex === i,
+                        })}
+                      >
+                        {display || value}
+                      </MenuItem>
+                    );
+                  })}
+              </MenuList>
+            </Paper>
+          ) : null}
```
Comment (MarkDKoslow) [inline components/dropdown-question/index.tsx]: Not a blocker - I think it'd improve the UX if we showed a message when the user's search term doesn't match any results. Something like "No results for 'xyz'. Please update your search query." Right now it looks like it shows a small horizontal bar 
Screen Shot 2019-07-22 at 1 26 28 PM

**Comment by mellogarrett** [comment]
@MarkDKoslow I went ahead and implemented a "Your search returned no matches" element as well as some other small updates if you could kindly re-review. Thanks!

File: components/dropdown-question/index.test.tsx [hunk 2]
```diff
@@ -140,4 +143,21 @@ describe('DropdownQuestionComponent', () => {
     fireEvent.change(getByLabelText(testQuestion.title), { target: { value: 'abc' } });
     expect(queryAllByRole('option').length).toBe(1);
   });
+
+  test('should show disabled option when search yields no results', () => {
```
Comment (MarkDKoslow) [inline components/dropdown-question/index.test.tsx]: Nice! 👏

**Comment by MarkDKoslow** [comment]
Looks great! Thanks for adding that 👏

**Pull request merged**